frontend/public/components/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
```

Comment:
I don't see the point in making a generic component out of a patternfly variant. This way we would end up with a lot of generic components in the public folder. I would prefer we use `Select` wherever we need it.

Reply:
made the changes... please review